server/eventlog_test.go

```diff
@@ -347,7 +350,66 @@ func TestParseSecureBootState(t *testing.T) {
 			t.Error("expected to see both WinProdPCA and ThirdPartyUEFI certs")
 		}
 	}
+}
+
+func TestParsingCELEventLog(t *testing.T) {
```

Comment:
consider testing an empty CEL and a CEL without any COS types (but other random TLVs)

Reply:
Added an empty CEL test, note an empty CEL can pass Verify with any PCR values as no record can be check against with.

Test with non COS type is tricky as currently we only have one type (CosTlv) that implements the Content interface, I can try to manually create a Record and manually generate it digests.

Reply:
Added the random TLV test. Also I realized we can read PCR value directly from the TPM to do the test.

Reply:
thanks!
small nit: maybe break these up into different tests as they are starting to be difficult to read
feel free to leave if you disagree.